src/guide/troubleshooting/installation-issues.md

```diff
@@ -134,3 +134,26 @@ toolchain installed. Then, run:
 ```bash
 $ rustup toolchain install stable
 ```
+
+## Troubleshooting Python toolchain
+
+When you install the Python Wheel package using pip on the "[client setup](../python#_1-iroha-2-client-setup)" step, you may encounter an error like:
+"iroha_python-*.whl is not a supported wheel on this platform".
+
+This error means that pip is outdated, so you need to update it.
+First of all, it is recommended to check your OS for updates and perform a system upgrade.
+
+If this doesn't work, you can try updating `pip` for your user directory.
+
+`python -m pip install --upgrade pip`
+
+Make sure that `pip` that is installed in your home directory by running `whereis pip` and checking if `/home/username/.local/bin/pip` is among the paths. You may need to update your shell's `PATH` variable otherwise.
```

Comment:
```suggestion
Make sure that this `pip` is the same `pip` that is installed in your home directory by running `type pip` and then double checking `whereis pip` to see if  `/home/`_username_`/.local/bin/pip` is among the paths. You may need to update your shell's `PATH` variable otherwise.
```